projector-server/src/main/kotlin/org/jetbrains/projector/server/ProjectorServer.kt

```diff
@@ -869,9 +872,9 @@ class ProjectorServer private constructor(
       private set
 
     const val ENABLE_PROPERTY_NAME = "org.jetbrains.projector.server.enable"
-    const val HOST_PROPERTY_NAME_OLD = "org.jetbrains.projector.server.host"
+    private const val HOST_PROPERTY_NAME_OLD = "org.jetbrains.projector.server.host"
     const val HOST_PROPERTY_NAME = "ORG_JETBRAINS_PROJECTOR_SERVER_HOST"
-    const val PORT_PROPERTY_NAME_OLD = "org.jetbrains.projector.server.port"
+    private const val PORT_PROPERTY_NAME_OLD = "org.jetbrains.projector.server.port"
```

Comment:
This is refactoring a bit unrelated. I don't mind applying it though. Could you please at least extract it to a separate commit in this PR?

Reply:
Done